docs/DafnyRef/Attributes.md

```diff
@@ -32,6 +33,19 @@ overrides those further away.
 For attributes with a single boolean expression argument, the attribute
 with no argument is interpreted as if it were true.
 
+** Migration notes: ** There is a new syntax for typed attributes that is being added: `@Attribute(...)`, and these attributes are going to be prefix attributes. For now, the new syntax works only as top-level declarations. When all previous attributes will be migrated, this section will be rewritten.
```

Comment:
I would include a quick code example of what using one of these attributes looks like, before the next paragraph that is talking more about implementation.